crucible-mir/CHANGELOG.md

```diff
@@ -28,6 +28,9 @@ This release supports [version
 * Type translation functions like `tyToRepr` now fail gracefully
   so that failed translations can be handled by upstream tooling
   instead of failing using `error`
+* Fix a bug in which static items with non-constant initializer expressions that
+  depend on static items with constant initializer expressions would fail to
+  translate.
```

Comment:
Small nitpick - they would translate fine (no `error` or `mirFail`), but the `static_initializer` function (which runs automatically before the test) would fail at run time due to reading from a pointer to uninitialized memory.